### DISCUSSION TITLE
de/2-02-kata-1/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#56] [General]
# de/2-02-kata-1/

https://ai.lernos.org/de/2-02-kata-1/

**Comment by tschaena** [comment]
Für komplette Neueinsteiger finde ich dieses erste Kata suboptimal. Es ist sehr theoretisch, konzeptionell und reflektiv was wohl die eine oder andere Person abschreckt, hier weiterzumachen. Eine Möglichkeit, wäre, dieses Kata mit Kata 2 zu tauschen und im Kata 2 für komplette Neuanfänger eine sehr konkrete unkritische Übung vorzugeben, z.B. mit ChatGPT und co. einen Text-Abschnitt aus Wikipedia in Bullet-points Form umschreiben oder übersetzen zu lassen. Ein kleiner Hinweis bezüglich Einhaltung möglicher Policies der Organisation würde für so eine Übung schon mal reichen.

**Reply by simondueckert** [reply]
War ein bisschen wie der Disclaimer in der ersten Livesession gedacht, wir wollen vermeiden, dass wir insbesondere Menschen in Organisationen auf "illegale Pfade" locken. Gerade der Hinweis auf ChatGPT würde genau das tun, das das Tool fast überall im Kontext der Organisation nicht erlaubt ist. Werden wir aber nochmal besprechen, danke für den Hinweis.

**Comment by mkmikado** [comment]
Lese Dir das Kapitel KI... --> <b>Lies</b> Dir...
...Schadecode zu erstellen.  --> ...<b>Schadcode</b> zu erstellen.

**Comment by olelai** [comment]
Auch ich finde dieses erste Kata nicht so gut. Klar, die Rahmenbedingungen müssen geklärt werden. Aber dieses Kata bremst die Lernenden aus. Sie wollen etwas Neues ausprobieren und bekommen statt dessen die ungeliebten Themen Datenschutz und Urheberrecht zu lesen. 

Könnte man nicht beides verbinden, z.B. mit einer Recherche zum  mit Perplexity.ai? Perplexity kann man ohne Konto benutzen, so dass im Zweifel nur die IP-Adresse und ein Browser-Fingerprint gespeichert sind. Z.B. könnten die TN mit Perplexity recherchieren, was sie zum Datenschutz, resp. Urheberrecht beachten müssen, wenn sie ein LLM nutzen. Habe ich eben gemacht und die Antworten sind gut.

So kommen die TN ins Tun.

**Reply by dieSabrina** [reply]
Ich verstehe den Gedanken - nur: Hat man als neugieriger Mensch der ja auch den MOOC gefunden hat und starten mag, nicht zumindest schon so etwas oder ähnliches gemacht und mit KIs herumprobiert? Ich fühle mich beim Starten umgekehrt eher abgeschreckt, wenn ich den Eindruck bekomme, das sehr niedrig eingestiegen wird. - Tja....ich weiß nicht, ist vermutlich auch viel Geschmacksache ;)

**Reply by olelai** [reply]
Volle Zustimmung, das hat mich auch abgeschreckt. Für mich startet der MOOC erst heute richtig. Vielleicht wäre es sinnvoll, Katas für Einsteiger und für etwas weiter Fortgeschrittene zu gestalten. Das ist doppelte Arbeit, ich weiß.

**Comment by MonacoFranke** [comment]
Könnt Ihr die Aufgaben der Kata jeweils nummerieren, dann ist es einfacher im Learncircle drüber zu sprechen

**Reply by SylwiaMiedza** [reply]
Das ist mir auch schon aufgefallen: Die Aufgaben nummerieren und gerne auch (fett? oder mit einem Icon) hervorheben.

**Comment by SylwiaMiedza** [comment]
Ich finde den Hinweis auf vorhandene Richtlinien sowie die Empfehlungen bei noch nicht vorhandenen Richtlinien richtig gut und gerade zu Anfang wichtig. 
Es ist ja auch sehr überschaubar, sodass man sich damit auch nicht ewig aufhält, diese aber gleich am Anfang nochmal vorgelegt bekommt. 

Es ist nicht immer gut, sofort ins Tun zu kommen. 
Erst lesen, dann machen. Es ist wie bei Gebrauchsanweisungen: manchmal ist es doch sinnvoll, sie vorher zu lesen :-)
